src/store/actions/todo.js

```diff
@@ -13,3 +17,10 @@ export const deleteTaskFromTodo = (taskId) => {
     payload: taskId,
   };
 };
+
+export const markTaskAsDone = (taskId) => {
```

Comment:
As all the actions will be primarily done on a Task, we can omit Task from the action names.

```suggestion
export const markAsDone = (taskId) => {
```